src/lt/objs/plugins.cljs

```diff
@@ -494,7 +494,7 @@
    (search-input this)])
 
 (defui source-button [plugin]
```

Comment:
No need to do the below in this PR, but just taking the opportunity to open a potential conversation:

One question I would like to generally propose to anyone reading is if we should change the code from `source` to `website`... such as the defui's `source-button` to `website-button`. Minor detail, but I feel this could potentially break custom themes. Probably best to simply open an issue to address it in the future.